client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx

```diff
@@ -0,0 +1,135 @@
+
+    React.useEffect(() => {
+        setCanCreateEvent(eventStartTime !== undefined && eventEndTime !== undefined);
+    }, [eventStartTime, eventEndTime]);
+
+    return (
+        <Dialog classes={{paper: classes.dialogPaper}} open={isOpen} maxWidth={false}>
+            <DialogTitle className={classes.dialogTitleWrapper}>
+                {newConactEventTitle}
+            </DialogTitle>
+            <DialogContent>
+                <Grid className={formClasses.form} container justify='flex-start'>
+                    <div className={formClasses.rowDiv}>
+                        <Grid item xs={3}>
+                            <Typography variant='caption' className={formClasses.fieldName}>
+                                סוג אתר:
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={9}>
+                            <Select
+                                value={placeType}
+                                onChange={(event: React.ChangeEvent<any>) => setPlaceType(event.target.value)}
+                                className={classes.placeTypeSelect}
+                            >
+                                {
+                                    selectData.map((placeName: string) => (
+                                        <MenuItem key={placeName} value={placeName}>{placeName}</MenuItem>
+                                    ))
+                                }
+                            </Select>
+                        </Grid>
+                    </div>
+                    <div className={formClasses.rowDiv}>
+                        <Grid item xs={3}>
+                            <Typography variant='caption' className={formClasses.fieldName}>
+                                משעה:
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={9}>
+                            <DatePick 
+                                datePickerType='time'
+                                value={eventStartTime || defaultTime}
+                                onChange={(event: React.ChangeEvent<HTMLInputElement>) => setEventStartTime(event.target.value as string)}
+                            />
+                        </Grid>
+                    </div>
+                    <div className={formClasses.rowDiv}>
+                        <Grid item xs={3}>
+                            <Typography variant='caption' className={formClasses.fieldName}>
+                                עד שעה:
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={9}>
+                            <DatePick 
+                                datePickerType='time' 
+                                value={eventEndTime || defaultTime}
+                                onChange={(event: React.ChangeEvent<HTMLInputElement>) => setEventEndTime(event.target.value)}/>
```

Comment:
its better to put prop functions in const or named function so they would be recreated on render